src/Jackett.Common/Definitions/ygg-api.yml

```diff
@@ -0,0 +1,184 @@
+        .Config.default_movie_query }}{{ end }}
+        &category_id=2183&category_id=2184&order_by={{ .Config.sort
+        }}&per_page=100
+      method: get
+      response:
+        type: json
+    - path: /torrents?page=1&q={{ if .Keywords }}{{ .Keywords }}{{ else }}{{
+        .Config.default_tv_query }}{{ end }}
+        &category_id=2184&category_id=2179&order_by={{ .Config.sort
+        }}&per_page=100
+      method: get
+      categories:
+        - 2184
+      response:
+        type: json
+    - path: /torrents?page=2&q={{ if .Keywords }}{{ .Keywords }}{{ else }}{{
+        .Config.default_tv_query }}{{ end }}
+        &category_id=2184&category_id=2179&order_by={{ .Config.sort
+        }}&per_page=100
+      method: get
+      categories:
+        - 2184
+      response:
+        type: json
+    - path: /torrents?page=1&q={{ if .Keywords }}{{ .Keywords }}{{ else }}{{
+        .Config.default_movie_query }}{{ end }}
+        &category_id=2183&category_id=2178&order_by={{ .Config.sort
+        }}&per_page=100
+      method: get
+      categories:
+        - 2183
+      response:
+        type: json
+    - path: /torrents?page=1&q={{ if .Keywords }}{{ .Keywords }}{{ else }}{{
+        .Config.default_anime_query }}{{ end }}
+        &category_id=2179&category_id=2178&order_by={{ .Config.sort
+        }}&per_page=100
+      method: get
+      categories:
+        - 2179
+        - 2178
+      response:
+        type: json
+    - path: /torrents?page=2&q={{ if .Keywords }}{{ .Keywords }}{{ else }}{{
+        .Config.default_anime_query }}{{ end }}
+        &category_id=2179&category_id=2178&order_by={{ .Config.sort
+        }}&per_page=100
+      method: get
+      categories:
+        - 2179
+        - 2178
+      response:
+        type: json
+  keywordsfilters:
+    - name: re_replace
+      args:
+        - "[\\[\\]()]"
+        - ""
+  rows:
+    selector: $
+    missingAttributeEqualsNoResults: true
+  fields:
+    title_normal:
+      selector: title
+    title_multilang:
+      text: "{{ .Result.title_normal }}"
+      filters:
+        - name: re_replace
+          args:
+            - (?i)\b(MULTI(?!.*(?:FRENCH|ENGLISH|VOSTFR)))\b
+            - "{{ .Config.multilanguage }}"
+    title:
+      text: "{{ if .Config.multilang }}{{ .Result.title_multilang }}{{ else }}{{
+        .Result.title_normal }}{{ end }}"
+    category:
+      selector: category_id
+    year:
+      selector: title
+      filters:
+        - name: regexp
+          args: (\b(19|20)\d{2}\b)
+    grabs:
+      selector: downloads
+      optional: true
+      default: 0
+    details:
+      selector: id
+      filters:
+        - name: prepend
+          args: https://yggapi.eu/torrent/
+    download:
+      selector: id
+      filters:
+        - name: prepend
+          args: https://yggapi.eu/torrent/
+        - name: append
+          args: /download?passkey={{ .Config.passkey }}
+    size:
+      selector: size
+    _quality:
```

Comment:
quality is not a standard field, in that it is not returned in torznab results.
so it is not necessary to extract it unless you plan to incorporate it into one of the other fields,
such as description for example. or you could just drop it altogether.